app/src/main/java/org/gdg/frisbee/android/fragment/SettingsFragment.java

```diff
@@ -101,36 +99,29 @@ protected Void doInBackground(Void... voids) {
                             }
 
                             GdgXHub client = App.getInstance().getGdgXHub();
-                            String token = GoogleAuthUtil.getToken(getActivity(), 
-                                    Plus.AccountApi.getAccountName(mGoogleApiClient), 
+                            String token = GoogleAuthUtil.getToken(getActivity(),
+                                    Plus.AccountApi.getAccountName(mGoogleApiClient),
                                     "oauth2: " + Scopes.PLUS_LOGIN);
 
                             if (!enableGcm) {
-                                client.unregisterGcm("Bearer " + token, new GcmRegistrationRequest(PrefUtils.getRegistrationId(getActivity())),
-                                        new Callback<GcmRegistrationResponse>() {
+                                client.unregisterGcm("Bearer " + token, new GcmRegistrationRequest(PrefUtils.getRegistrationId(getActivity())))
+                                        .enqueue(new Callback<GcmRegistrationResponse>() {
                                             @Override
-                                            public void success(GcmRegistrationResponse gcmRegistrationResponse, retrofit.client.Response response) {
+                                            public void onSuccessResponse(GcmRegistrationResponse response) {
                                                 PrefUtils.setGcmSettings(getActivity(), false, null, null);
                                             }
-
-                                            @Override
-                                            public void failure(RetrofitError error) {
-                                                Timber.e(error, "Fail");
```

Comment:
Error logging like these are removed. All of the errors are now logged automatically. 
It is a network connection or timeout error it only debug logs it. If it is a server error, it error logs it and thus sends it to Crashlytics.